### DISCUSSION TITLE
level alias

### DISCUSSION THREAD

**Discussion by chengm349** [gabime/spdlog#2238] [General]
[18:03:45 693012] [15955] [critical] Support for int: 42;  hex: 2a;  oct: 52; bin: 101010

I remember there is a simple way to change [critical] to [fatal] for example.

**Comment by tt4g** [comment]
You can use `SPDLOG_LEVEL_NAMES` and `SPDLOG_SHORT_LEVEL_NAMES` macro.

https://github.com/gabime/spdlog/blob/v1.9.2/include/spdlog/tweakme.h#L89-L101

Example:

```cpp
#define SPDLOG_LEVEL_NAMES \
    { spdlog::string_view_t("trace", 5)     \
    , spdlog::string_view_t("debug", 5)     \
    , spdlog::string_view_t("info", 4)      \
    , spdlog::string_view_t("warning", 7)   \
    , spdlog::string_view_t("error", 5)     \
    , spdlog::string_view_t("fatal", 5)  \
    , spdlog::string_view_t("off", 3) }

#include "spdlog/spdlog.h"

int main(void)
{
    spdlog::info("Hello, World");

    return 0;
}
```

**Reply by chengm349** [reply]
works and thanks.

**Comment by chengm349** [comment]
seems SPDLOG_LEVEL_NAMES does not like -DSPDLOG_COMPILED_LIB. If I turn off the flag, I got fatal. If I turn on -DSPDLOG_COMPILED_LIB, still critical.
